deploy-service/common/src/main/java/com/pinterest/deployservice/handler/PingHandler.java

```diff
@@ -195,7 +195,9 @@ boolean canDeploy(EnvironBean envBean, String host, AgentBean agentBean) throws
         // TODO use ecache to optimize
         // Make sure we do not exceed allowed number of concurrent active deploying agent
         String envId = envBean.getEnv_id();
-        long parallelThreshold = envBean.getMax_parallel();
+        long totalHosts = environDAO.countTotalCapacity(envBean.getEnv_id(), envBean.getEnv_name(),
```

Comment:
If you only want the active hosts, you can use this query: agentDAO.getAllByEnv(envBean.getEnv_id()).size().